DependencyInjection/Compiler/StoragePass.php

```diff
@@ -22,8 +22,10 @@
 class StoragePass implements CompilerPassInterface
 {
     const FILE_OVERLAY_SERVICE_ID = 'pb_sulu_storage.file.overlay';
-    const FILE_RESOLVER_SERVICE_ID = 'pb_sulu_storage.file_resolver';
-    const FILESYSTEM_PROVIDER_SERVICE_ID = 'pb_sulu_storage.filesystem_provider';
+    const STORAGE_FILE_RESOLVER_SERVICE_ID = 'pb_sulu_storage.storage_file_resolver';
+    const STORAGE_FILESYSTEM_PROVIDER_SERVICE_ID = 'pb_sulu_storage.storage_filesystem_provider';
+    const FORMAT_CACHE_FILE_RESOLVER_SERVICE_ID = 'pb_sulu_storage.format_cache_file_resolver';
+    const FORMAT_CACHE_FILESYSTEM_PROVIDER_SERVICE_ID = 'pb_sulu_storage.format_cache_filesystem_provider';
```

Comment:
This would be a breaking change if anyone uses these services. I could preserve the original names and introduce two new ones instead, let me know if you want me to do that.

Reply:
Alright, reverted the service name changes for the storage file resolver and filesystem provider. I've decided to keep the new constant names instead of adding a comment. Is that fine by you?